subworkflows/nf-core/fasta_newick_epang_gappa/meta.yml

```
@@ -0,0 +1,82 @@
+name: "fasta_newick_epang_gappa"
+description: Run phylogenetic placement with a number of query sequences plus a reference alignment and phylogeny. Used in nf-core/phyloplace.
+keywords:
+  - phylogenetic placement
+  - phylogenetics
+  - alignment
+  - fasta
+  - newick
+modules:
+  - hmmer/hmmbuild
+  - hmmer/hmmalign
+  - hmmer/eslalimask
+  - hmmer/eslreformat
+  - epang/place
```

is here `mafft` missing?
```suggestion
  - mafft
  - epang/place
```